src/qibolab/backends.py

```diff
@@ -50,6 +51,25 @@ def __init__(self, platform):
         }
         self.compiler = Compiler.default()
 
+    @property
+    def qubits(self) -> list[Union[str, int]]:
+        return list(self.platform.qubits.keys())
```

Comment:
```suggestion
        return list(self.platform.qubits)
```
`list()` is returning a list out of an iterable object, accumulating its elements.

The iteration over a `dict` is actually iterating its keys, so `.keys()` is just redundant (not harmful).